beluga/include/beluga/tuple_vector.hpp

```diff
@@ -203,19 +203,11 @@ class TupleContainer<InternalContainer, std::tuple<Types...>> {
   }
 
   [[nodiscard]] constexpr auto all() const {
-    return std::apply(
-        [](auto&... containers) {  //
```

Comment:
@nahueespinosa meta: now I wonder how this managed to compile. Did the implicit constness of `sequences_` not propagate to its elements?

Reply:
It did, the `const` I added here does nothing, but I thought it would make sense to add it. Now that you say it, I think I better just remove it. Done in https://github.com/Ekumen-OS/beluga/pull/288/commits/9d114352e16cb0cd14f1124c758e16e42c7b78bd.